flexget/components/sites/sites/piratebay.py

```diff
@@ -130,76 +137,80 @@ def search(self, task, entry, config=None):
         if not isinstance(config, dict):
             config = {}
         self.set_urls(config.get('url', URL))
-        sort = SORT.get(config.get('sort_by', 'seeds'))
-        if config.get('sort_reverse'):
-            sort += 1
+        sort = SORT.get(config.get('sort_by', 'default'))
+        sort_reverse = bool(config.get('sort_reverse', 'True'))
         if isinstance(config.get('category'), int):
             category = config['category']
         else:
             category = CATEGORIES.get(config.get('category', 'all'))
-        filter_url = '/0/%d/%d' % (sort, category)
+        filter_url = '&cat=%s' % category
 
         entries = set()
         for search_string in entry.get('search_strings', [entry['title']]):
             query = normalize_unicode(search_string)
 
             # TPB search doesn't like dashes or quotes
-            query = query.replace('-', ' ').replace("'", " ")
+            # query = query.replace('-', ' ').replace("'", " ")
```

Comment:
forgot comment?

Reply:
The new API appears to accept all characters, even Unicode. This code is now redundant.